crates/matrix-sdk-crypto/src/backups/keys/decryption.rs

```diff
@@ -181,10 +191,19 @@ impl BackupDecryptionKey {
         PkDecryption::from_bytes(self.inner.as_ref())
     }
 
+    fn calculate_mac_key(&self) -> HmacSha256Key {
```

Comment:
Can't we refactor things a bit so we don't have this piece of code in two places?

Reply:
I'm not sure which other place you're talking about.

Reply:
Oh, both methods are called `calculate_mac_key()` but they indeed do slightly different things. I guess that's fine then.